book/validation.rst

```diff
@@ -391,10 +395,13 @@ constraint, have several configuration options available. Suppose that the
 
             public static function loadValidatorMetadata(ClassMetadata $metadata)
             {
-                $metadata->addPropertyConstraint('gender', new Choice(array(
```

Comment:
It introduces longer code examples, but doesn't add much value imo (if any at all).